core/arch/arm/plat-zynqmp/drivers/aes.h

```diff
@@ -0,0 +1,23 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (C) Foundries Ltd. 2021 - All Rights Reserved
+ * Author: Jorge Ramirez <jorge@foundries.io>
+ */
+
+#ifndef AES_H_
+#define AES_H_
+
+#include <types_ext.h>
+#include <tee_api_types.h>
+
+enum csu_key {
+	CSU_AES_KEY_SRC_KUP,
```

Comment:
Better set first one with `CSU_AES_KEY_SRC_KUP = 0` to be exactly clear that this needs to be zero as enum value goes right to register.

Reply:
yes